components/textfield/index.css

```diff
@@ -304,12 +274,8 @@
 		grid-column: 1 / span 1;
 	}
 
-	.spectrum-Textfield--quiet & {
-		margin-block-end: var(--mod-textfield-label-spacing-block-quiet, var(--spectrum-textfield-label-spacing-block-quiet));
-	}
-
 	.is-disabled & {
-		color: var(--spectrum-textfield-text-color-disabled);
+		color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)));
```

Comment:
There were a few disabled selectors like this one where there was no `--mod` being offered, I added one back in since I didn't see a reason not to.